c2corg_api/models/association.py

```diff
@@ -381,5 +381,6 @@ def _get_associations_to_add(new_associations, current_associations):
                    'outings', 'books', 'xreports'},
     AREA_TYPE: {'images'},
     BOOK_TYPE: {'routes', 'articles', 'images', 'waypoints'},
-    XREPORT_TYPE: {'routes', 'outings', 'articles', 'images'}
+    XREPORT_TYPE: {'routes', 'outings', 'articles', 'images', 'waypoints',
```

Comment:
I do not think WP should be interesting (confusing witg I/A geolocalization). If needed, we can do **later** a join with outings or route associated.

Articles associations could be interesting.